src/components/CategoryFilter.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import categories from '../config'
```

Comment:
so when you do like this, it picks up the `index.js` in the `config` folder ? 😊

Reply:
That's right!